### ISSUE TITLE
Include user who submitted package in API responses

### ISSUE THREAD

**Issue by AdmiringWorm** [chocolatey/chocolatey.org#901]
The current API for getting a single package details version does not include the user that submitted that specific package version.

I am requesting that this bit of information to be added to the API.
The reason I would like this to be included is for some package request status checks I am working on in the regards of the [chocolatey-package-requests](https://github.com/chocolatey-community/chocolatey-package-requests), where it would be useful to have this bit of information to verify that a new user that has picked up a request is also the user that submitted the latest version.

**Comment by ferventcoder** [comment]
Agreed, would be good to have this information. I don't see any privacy concerns as this is public information on the site.
